service/controller/app/v1/resource/status/create.go

```diff
@@ -32,6 +32,21 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	if errors.IsNotFound(err) {
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not find chart %#q in namespace %#q", name, r.chartNamespace))
 		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+
+		currentCR, err := r.getCurrentCR(cr)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		currentCR.Status = v1alpha1.AppStatus{}
+
+		_, err = r.g8sClient.ApplicationV1alpha1().Apps(cr.Namespace).UpdateStatus(currentCR)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("status clear for app %#q in namespace %#q", name, r.chartNamespace))
```

Comment:
There should be an opening and closing log, just like everywhere else in the operators. The wording should also be reworked.